### ISSUE TITLE
Validation Dataset

### ISSUE THREAD

**Issue by wyf6699** [kylekma/X2CT#24]
Hello, I am very interested in your project, but I have some questions that I need you to answer. In general, network training requires training set, validation set and test set, but I only see training set and test set in the paper and code, where is the validation set.
